source/comply/legal-hold.rst

```diff
@@ -43,86 +43,65 @@ Establish a policy for when to implement a legal hold. This should be developed
 
 .. note::
 
-    - For legal hold, only XML format is supported at this time. 
-    - Support for additional file formats, such as CSV or EML, are under consideration.
-
-3 - Leverage compliance API in your DLP tool of choice
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+  Only Amazon S3 is supported for Mattermost legal holds at this time, for increased reliability, compliance, and automation capabilities. Support for additional file storage options such as MinIO or local file storage are under consideration. Learn more about file storage configuration options in our :ref:`product documentation <configure/environment-configuration-settings:file storage>`.
 
-Connect it directly to the Amazon S3 bucket where compliance exports are stored. Our recommended DLP tool of choice is `Smarsh <https://www.smarsh.com/>`_. Learn more about their legal hold & e-discovery processes in their `documentation <https://www.smarsh.com/platform/enterprise/discovery>`_.
-
-You may optionally also use other tools that connect with Amazon S3, such as Active360, AwareHQ, Onna, or Trellix, though our team will be able to provide limited support using these tools.
-
-4 - (Optional) Configure a data retention policy
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-This is typically implemented alongside legal hold, though is not required.
+(Optional) Configure a data retention policy
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-1. In Mattermost, go to **System Console > Compliance > Data Retention Policies**.
-2. Set either a **Global retention policy** that applies to all messages and files older in your system, or a **custom retention policy** for specific teams or channels.
-3. Learn more about data retention configuration options in our :doc:`product documentation </comply/data-retention-policy>`.
+You can optionally configure a :doc:`data retention policy </comply/data-retention-policy>` for the legal hold data you want to retain and delete to align with compliance requirements.
 
-Step 4: Identify custodians
+Step 4: Create a legal hold
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Select the individuals or user groups that you want to place on legal hold.
+In Mattermost, create a legal hold by completing the following steps:
 
-Step 5: Initiate legal hold
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+1. Go to **System Console > Plugins > Legal Hold Plugin**, and select **Create new**.
+2. Enter a name for the legal hold.
+3. Specify the custodian user names of individuals or user groups that you want to place on legal hold.
+4. (Optional) You can choose to exclude public channels, if preferred.
```

Comment:
The default is to exclude public channels, so this wording may be more clear reversed. `(Optional) You can choose to include public channels, if preferred`.